File: CarUnit_be/src/main/java/com/ilCary/CarUnit/models/Comune.java
package com.ilCary.CarUnit.models;

import lombok.*;

import javax.persistence.*;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@Builder
@Entity
@Table(name = "municipalities")
public class Comune {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String comune;
    private String provincia;
    private String cap;
    private String regione;

}

File: CarUnit_fe/CarUnit/src/app/models/Comune.ts
export class Comune {
  id:number;
  comune:string;
  provincia:string;
  cap:string;
  regione:string;

  constructor(id:number,comune:string,provincia:string,cap:string,regione:string){
    this.id = id;
    this.comune = comune;
    this.provincia = provincia;
    this.cap = cap;
    this.regione = regione;
  }
}
